client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -90,11 +91,15 @@ const BrewItem = createClass({
 		</a>;
 	},
 
-	renderGoogleDriveIcon : function(){
-		if(!this.props.brew.googleId) return;
+	renderStorageIcon : function(){
+		if(!this.props.brew.googleId) return <span title='Internal Homebrewery Database'>
```

Comment:
Instead of the negative, `(!this.props.brew.googleId)`, can we reorder the logic so we can use the positive case (`if(this.props.brew.googleId)`?

Reply:
Absolutely - as you can see, I was just throwing in some output for local storage, rather than the previous nothing.